### ISSUE TITLE
CSV loading tutorial examples fail

### ISSUE THREAD

**Issue by mccalluc** [Caleydo/caleydo.github.io#109]
I just pushed the changes to use the bundle on s3, so the smiley-face examples are working, but
the latter examples on http://www.caleydo.org/documentation/tutorial/web_bundle/ which rely on CSV loading are not working. There are JS errors in the console on both FF and Chrome, though the actual problem isn't obvious to me. The HTTP request for the CSV seems fine.

@thinkh : Are the files on S3 exactly the same ones as were checked in previously? If that code didn't change, I'm not sure what else to attribute the change to.

**Comment by mstreit** [comment]
The version that is hosted on S3 is from the latest build.

**Comment by mccalluc** [comment]
Ok, that helps to know: We already knew that there were differences between the latest build, and the version that had been checked in. This narrows it down a little: The examples with the inline data work fine, so it's just a problem with file fetching or CSV parsing.

**Comment by mstreit** [comment]
@thinkh Do you know why the current bundle version behaves differently?

**Comment by thinkh** [comment]
> The version that is hosted on S3 is from the latest build. 

For clarification: I downloaded the files from the github bundle_lib release and uploaded it to S3 twice (latest and v0.0.1). These should be identical at the moment.

The download of the CSV file is working as expected.

![tutorial](https://cloud.githubusercontent.com/assets/5851088/18316095/a191d27e-7519-11e6-8023-e28dbef4aa55.png)

However, an error occured in the library, but it's hard to debug with the minified version.
@mccalluc: Could you switch it to the non-minified version for debugging purpose?

**Comment by thinkh** [comment]
I fixed the web_bundle tutorial now. I changed the parseMatrix function that the given data array must contain the `col_id` as first element in the array and the first element of each row defines then the `row_id` (used in example 3+4).
However, if you pass an array with row_id and col_id as second and third parameter, the data array should contain only the data itself (as you did in example 0-2).

I updated S3 to the most recent version
